packages/styleguide/src/examples/pages/aui-avatar/aui-avatar.page.html

```diff
@@ -13,7 +13,7 @@ <h2 class="h3 u-margin-bottom">Avatar</h2>
 <div class="u-margin-bottom">
   <aui-code-snippet [codeSnippet]="example2"></aui-code-snippet>
   <div class="u-margin-bottom-xs">
-    <aui-avatar icon="fa fa-user" size="L" title="A user icon avatar"></aui-avatar>
+    <aui-avatar icon="ai-alarm-bell" size="L" title="An alarm bell icon"></aui-avatar>
```

Comment:
Some labels are dutch some are English when do you use which. Is there a rule?

Reply:
It works like this: the default values are in Dutch, since most (if not all) the apps made with the lib are in Dutch, but the documentation is in English. I actually like this setup, because we can test translations in advance and the developers don't need to do the extra work.